### PR TITLE
APP-1665 [Deployment GroupID] Configure log4j

### PR DESCRIPTION
**Pull request by robson-sanchez**
- Remove spring-boot-starter-logging dependency (Logback)
- Include spring-boot-starter-log4j2 dependency (Log4j2)
- Include disruptor dependency to support async logger
- Configure log4j settings for all applications
- Update startup script template for all applications

### CODE REVIEW

File: helpdesk-application/pom.xml [hunk 1]
```diff
@@ -25,6 +25,22 @@
...
+        </dependency>
+        <dependency>
+            <groupId>org.springframework.boot</groupId>
+            <artifactId>spring-boot-starter-log4j2</artifactId>
+            <version>${spring.boot.version}</version>
+        </dependency>
+        <dependency>
+            <groupId>com.lmax</groupId>
+            <artifactId>disruptor</artifactId>
+            <version>3.3.6</version>
```
Comment (campidelli) [inline helpdesk-application/pom.xml]: Please use a variable to define versions.

File: helpdesk-bot/pom.xml [hunk 2]
```diff
@@ -36,6 +36,22 @@
...
+        </dependency>
+        <dependency>
+            <groupId>org.springframework.boot</groupId>
+            <artifactId>spring-boot-starter-log4j2</artifactId>
+            <version>${spring.boot.version}</version>
+        </dependency>
+        <dependency>
+            <groupId>com.lmax</groupId>
+            <artifactId>disruptor</artifactId>
+            <version>3.3.6</version>
```
Comment (campidelli) [inline helpdesk-bot/pom.xml]: Please use a variable to define versions.

File: helpdesk-service/helpdesk-service-server/pom.xml [hunk 3]
```diff
@@ -35,6 +35,22 @@
...
+        </dependency>
+        <dependency>
+            <groupId>org.springframework.boot</groupId>
+            <artifactId>spring-boot-starter-log4j2</artifactId>
+            <version>${spring.boot.version}</version>
+        </dependency>
+        <dependency>
+            <groupId>com.lmax</groupId>
+            <artifactId>disruptor</artifactId>
+            <version>3.3.6</version>
```
Comment (campidelli) [inline helpdesk-service/helpdesk-service-server/pom.xml]: Please use a variable to define versions.

**Review by campidelli — CHANGES_REQUESTED** [review]

**Review by crepache — APPROVED** [review]
LGTM

**Review by crepache — APPROVED** [review]
LGTM

**Pull request merged**